### PR TITLE
20 correctly handle spaces in paths, add test, fix release notes

### PR DESCRIPTION
**Pull request by btimbermont**
I went through the code and fixed all parts of the code that handle paths: not just pull, also install and ls.

I added new test code, and created a new ticket to transfer all test code to this new test class.

### CODE REVIEW

**Review by TeusSloot — APPROVED** [review]

File: adb_pywrapper/adb_device.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import subprocess
 from os import makedirs
 from os.path import basename, isfile
+from shlex import quote
```
Comment (rdvdijk) [inline adb_pywrapper/adb_device.py:4]: Just checking: `shlex` is part of the Python stdlib?

Comment (btimbermont) [reply adb_pywrapper/adb_device.py:4]: It is :)

**Review by Carly-1 — APPROVED** [review]
Looks good :)

File: test/test_adb_device.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        self.completed_adb_process = Mock(CompletedProcess)
+        self.stdout = stdout
+        self.stderr = stderr
+        self.success = success
+
+
+class TestAdbDevice(unittest.TestCase):
+
+    @parameterized.expand([
+        ('/sdcard/test.txt', './foo', 'pull /sdcard/test.txt ./foo'),
```
Comment (Marcel-NFI) [inline test/test_adb_device.py:26]: Could you make is more clear what the testcases are? Maybe add a comment or work the testcase into the case. E.g. 'this/is/a/case/with /a/space'

Comment (Marcel-NFI) [reply test/test_adb_device.py:26]: Maybe add a reference to the testcase for this bug?

**Pull request merged**